R/mean-model-workflow.R

```diff
@@ -91,9 +92,9 @@ MeanModelWorkflow <- R6::R6Class(
       appendices <- appendices[file.exists(appendices)]
       if (length(appendices) > 0) {
         mergeMarkdowndFiles(appendices, self$reportFileName, logFolder = self$workflowFolder)
-        actionToken2 <- re.tStartAction(actionType = "ReportGeneration", actionNameExtension = "runWorkflow")
+        # actionToken2 <- re.tStartAction(actionType = "ReportGeneration", actionNameExtension = "runWorkflow") #change delete

```

Comment:
@ju-rgen Here and everywhere where you added _#change delete_Can you please just remove commented lines?